beast/observationmodel/ast/make_ast_input_list.py

```diff
@@ -313,7 +313,7 @@ def pick_models(
         vega_f, vega_flux, lamb = v.getFlux(filters)
 
     # gridf = h5py.File(sedgrid_fname)
```

Comment:
We might want to remove the 3 lines including 'gridf' in this code that are currently all commented. I think we don't use 'gridf' anymore.